.github/workflows/ui-test-coverage.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
```

Comment:
Do we need this as a separate action? I would suggest having this part of one of the existing test workflows. We can update/enhance those workflows to be a single unit testing workflow.

More actions = More workers = More money = More spin up time = Longer pre-flight checks

Reply:
Moved it to single github action